src/PropertyDetail.js

```diff
@@ -53,7 +60,7 @@ export default function PropertyDetail() {
               <Typography variant={"body1"} component={"p"} gutterBottom><Link target="_blank" rel="noreferrer" href={`http://maps.google.com/?q=${encodedPropertyAddress}`}>View On Map</Link></Typography>
               <Typography variant={"h6"} gutterBottom>Condition/Quality</Typography>
               <Typography variant={"body1"} component={"p"}>Condition: {selectedProperty.CONDITION.trim() === "" ? 'Unknown' : selectedProperty.CONDITION}</Typography>
-              <Typography variant={"body1"} component={"p"} gutterBottom>Quality: {selectedProperty.QUALITY.trim() || 'Unknown'}</Typography>
+              <Typography variant={"body1"} component={"p"} gutterBottom>Quality: {selectedProperty.QUALITY.trim() === "" ? 'Unknown' : selectedProperty.QUALITY}</Typography>
```

Comment:
If the value of QUALITY is `""`, I would have expected: `QUALITY.trim()` to be falsey and end up with the same value being displayed.
I suspect I am missing something.

Reply:
good point.  I modified this to match what I did above it with {{CONDITION}}.  Maybe both actually worked.  I will test and go with the `||` if that one works since it is cleaner.